src/_guides/language/sound-problems.md

```diff
@@ -153,27 +150,40 @@ error • The method 'add' isn't defined for the class 'Iterable' at lib/bounded
 
 While the [List][] type has an `add()` method, [Iterable][] does not.
 
-#### Fix
+#### Fix: Specify type arguments or fix downstream errors
 
 In Dart 1.x, when a generic class is instantiated without explicit type
 arguments, `dynamic` is assumed. That is why, in the code excerpt above, `c` is
 of type `dynamic` and no error is reported for `c.add()`.
 
 In Dart 2, when a generic class is instantiated without explicit type arguments,
-each type parameter defaults to its type bound (`Interface` in this example) if
+each type parameter defaults to its type bound (`Iterable` in this example) if
 one is explicitly given, or `dynamic` otherwise.
 
 You'll need to approach fixing such errors on a case-by-case basis. It helps to
-have a good understanding of the original design intent. In this case, you might
-realize that there is a missing call to `toList()`:
+have a good understanding of the original design intent.
+
+Explicitly passing type arguments is one way to help identify type errors. For
+example, by specifying `List` as a type argument, the analyzer let's you know
```

Comment:
There are grammar and punctuation issues in this line (grammatically, the analyzer specifies List, and there's a punctuation issue with "let's"). How about this instead:

example, if you change the code to specify `List` as a type argument, the analyzer can detect

Reply:
Yes, much better. Done.